src/main/resources/properties/suite/web/suite.properties

```diff
@@ -1,3 +1,4 @@
 bdd.story-loader.batch-1.resource-location=story/web
 bdd.story-loader.batch-1.resource-include-patterns=*.story
+bdd.story-loader.batch-1.resource-exclude-patterns=login.story
```

Comment:
Please don't exclude specific stories from suite properties execution.
You can exclude them in local overriding.properties file